po/nl.po

```diff
@@ -240,6 +240,87 @@ msgstr "Annuleren"
 msgid "Warning"
 msgstr "Waarschuwing"
 
+msgid "Monday"
+msgstr "maandag"
+
+msgid "Tuesday"
+msgstr "dinsdag"
+
+msgid "Wednesday"
+msgstr "woensdag"
+
+msgid "Thursday"
+msgstr "donderdag"
+
+msgid "Friday"
+msgstr "vrijdag"
+
+msgid "Saturday"
+msgstr "zaterdag"
+
+msgid "Sunday"
+msgstr "zondag"
+
+msgid "Start (h/m):"
+msgstr "Begonnen op/om:"
+
+msgid "End (h/m):"
+msgstr "Afgerond op/om:"
+
+msgid "Scheduler enabled"
+msgstr "Planning ingeschakeld"
+
+msgid "Scheduler"
+msgstr "Planning"
+
+msgid "Enabled"
+msgstr "Ingeschakeld"
+
+msgid "Start downloading in these times"
+msgstr "Downloaden op bepaalde tijdstippen"
+
+msgid "Stop downloading in these times"
+msgstr "Downloaden onderbreken op bepaalde tijdstippen"
+
+msgid "Add Timespan"
+msgstr "Tijdspanne toevoegen"
+
+msgid "Seeding"
+msgstr "Bezig met seeden…"
+
+msgid "Failed"
+msgstr "Mislukt"
+
+msgid "Completion Options"
+msgstr "Afrondacties"
+
+msgid "Exit on Completion"
+msgstr "Afsluiten na afronden"
+
+msgid "Varia is about to quit."
+msgstr "Varia staat op het punt om te worden afgesloten."
+
+msgid "Preferences"
+msgstr "Voorkeuren"
+
+msgid "Other"
+msgstr "Overig"
+
+msgid "Basic Settings"
+msgstr "Algemene voorkeuren"
+
+msgid "Advanced Settings"
+msgstr "Geavanceerde voorkeuren"
+
+msgid "Use cookies.txt"
+msgstr "cookies.txt gebruiken"
+
+msgid "Import cookies.txt"
+msgstr "cookies.txt importeren"
+
+msgid "Remove cookies.txt"
+msgstr "cookies.txt verwijderenhttps://github.com/Vistaus/varia.git"
```

Comment:
Here the string has your fork's GitHub link, so that needs to be removed.

Reply:
I have no idea how that happened, lol. But thanks for noticing. 🙂 I just removed it.